catalog/core/catalog-core-commands/src/test/java/org/codice/ddf/commands/catalog/RemoveCommandTest.java

```diff
@@ -91,6 +101,39 @@ protected SolrCacheMBean getCacheProxy() {
         verify(mbean, times(1)).removeById(idsArray);
     }
 
+    @Test
+    public void testOverMaxItemList() throws Exception {
```

Comment:
For readability, try to separate inputs to test method from declarations. This can be done as mentioned in another of my comments about restructuring into class fields and setup method.